pilot/pkg/networking/core/v1alpha3/listener_address.go

```diff
@@ -39,6 +39,7 @@ const (
 var (
 	wildCards  map[model.IPMode][]string
 	localHosts map[model.IPMode][]string
+	ptBindIPs  map[model.IPMode][]string
```

Comment:
what is pt?

Reply:
Hi @hzxuzhonghu, `pt` means `passThrough`

Reply:
Can you just write passthrough then? Should be simpler to understand.

Reply:
Hi @howardjohn & @hzxuzhonghu, I changed the variable's name